src/interp/istream.cc

```diff
@@ -529,6 +531,8 @@ Instr Istream::Read(Offset* offset) const {
     case Opcode::GlobalSet:
     case Opcode::LocalSet:
     case Opcode::LocalTee:
+    case Opcode::InterpLocalSetRef:
```

Comment:
we think a better way would be to have an `InterpPushRefs` used alongside `InterpAlloca`, then we can avoid the `SetRef` entirely...